tests/neuro_san/internals/run_context/langchain/llms/test_llm_factory.py

```diff
@@ -10,154 +10,27 @@
 #
 # END COPYRIGHT
 
-from typing import Any
 from typing import Dict
+from typing import Type
 
-import json
+from neuro_san.internals.run_context.langchain.llms.llm_policy import LlmPolicy
+from neuro_san.internals.run_context.langchain.llms.standard_langchain_llm_factory import StandardLangChainLlmFactory
+from neuro_san.internals.run_context.langchain.llms.openai_llm_policy import OpenAILlmPolicy
 
-from langchain_core.language_models.base import BaseLanguageModel
-from langchain_openai.chat_models.base import ChatOpenAI
 
-from neuro_san.internals.run_context.langchain.llms.client_policy import ClientPolicy
-from neuro_san.internals.run_context.langchain.llms.langchain_llm_factory import LangChainLlmFactory
-from neuro_san.internals.run_context.langchain.llms.langchain_llm_resources import LangChainLlmResources
-from neuro_san.internals.run_context.langchain.llms.openai_client_policy import OpenAIClientPolicy
-
-
-class TestLlmFactory(LangChainLlmFactory):
+class TestLlmFactory(StandardLangChainLlmFactory):
```

Comment:
The TestLlmFactory can be drastically simplified now by inheriting from StandardLangChainLlmFactory.